code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/impl/ServiceMetadataInfoFactory.java

```diff
@@ -210,16 +211,19 @@ protected void getSidekicksInfo(Service service, List<String> sidekicks, List<St
         }
     }
 
-    protected void getLinksInfo(Map<Long, Service> idToService, Service service, Map<String, String> links) {
-        List<? extends ServiceConsumeMap> consumedMaps = consumeMapDao.findConsumedServices(service.getId());
+    protected void setLinksInfo(Map<Long, Map<String, ServiceMetaData>> services,
```

Comment:
super tiny, but looking at the method signature, I wouldn't have known it modified the contents of serviceMD.
But possibly rename to setLinksInfoInServiceMetaData() or setServiceMetaDataLinks()?